packages/cli-e2e/src/cli.smoke.spec.ts

```diff
@@ -23,13 +23,18 @@ describe('Qwik UI CLI Smoke test', () => {
 
   it('should be installed and add the button file', () => {
     execSync(
-      'npx -y qwik-ui@0.0.0-e2e init --e2e --projectRoot / --uiComponentsPath "src/components/ui" --rootCssPath "src/global.css" --installTailwind --style "simple" --borderRadius "0" --primaryColor "cyan-600" --components=button',
+      'npx -y qwik-ui@e2e init --e2e --projectRoot / --uiComponentsPath "src/components/ui" --rootCssPath "src/global.css" --installTailwind --style "simple" --components=button',
       {
         cwd: projectDirectory,
-        stdio: 'inherit',
         env: process.env,
+        stdio: 'inherit',
       },
     );
+    execSync('npx -y qwik-ui@e2e add button', {
+      cwd: projectDirectory,
+      env: process.env,
+      stdio: 'inherit',
+    });
     const buttonIsInTheRightPlace = existsSync(
```

Comment:
@shairez 
in a separate PR I'll remove the code that hints that it's possible to provide component names in the `init` phase 🙏

Reply:
but this is not the init phase no?

Reply:
It's not, but in the test, before my change, we had the components flag with the init command